extension-style/src/main/java/com/mapbox/maps/extension/style/light/generated/LightExt.kt

```diff
@@ -24,4 +25,11 @@ fun StyleInterface.setLight(light: StyleContract.StyleLightExtension) {
   light.bindTo(this)
 }
 
+/**
+ * Removes light from style if it was set.
+ */
+fun StyleInterface.removeLight() {
+  setStyleLight(Value.nullValue())
+}
```

Comment:
@kiryldz did you verify that this works? I'm getting an error with "light must be an object"

Reply:
Terrain seems fine though